editor/karma-setup.js

```diff
@@ -18,6 +18,7 @@ ReactTestingLibrary.configure({
   },
 })
 
+// this must run before importing the editor-entry-point-imports module
```

Comment:
This comment should be duplicated in `karma-base.conf.js`

Reply:
oh!

Reply:
pushed